Nightly is failing in `phpstan`

The [nightly](https://drone.owncloud.com/owncloud/richdocuments/2422/2/4) is failing with
```bash
1 package you are using is looking for funding.
Use the `composer fund` command to find out more!
php -d zend.enable_gc=0 vendor-bin/phpstan/vendor/bin/phpstan analyse --memory-limit=4G --configuration=./phpstan.neon --no-progress --level=5 appinfo lib
 ------ ----------------------------------------------------------------------------- 
  Line   lib/FileService.php                                                          
 ------ ----------------------------------------------------------------------------- 
         Ignored error pattern #Method                                                
         Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface::dispatch\(\)  
         invoked with 2 parameters, 1 required.# in path                              
         /var/www/owncloud/server/apps/richdocuments/lib/FileService.php was          
         not matched in reported errors.                                              
 ------ ----------------------------------------------------------------------------- 


 [ERROR] Found 1 error                                                          

```

Related core issue: https://github.com/owncloud/core/issues/39630

--- a/phpstan.neon
+++ b/phpstan.neon
@@ -12,6 +12,3 @@ parameters:
     -
       message: '#^Dead catch [^"]* is never thrown in the try block.$#'
       path: lib/Controller/WopiController.php  
-    -
-      message: '#Method Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface::dispatch\(\) invoked with 2 parameters, 1 required.#'
-      path: lib/FileService.php

